src/status_im/constants.cljs

```diff
@@ -120,6 +120,7 @@
 
 (def ^:const min-password-length 6)
 (def ^:const max-group-chat-participants 20)
+(def ^:const max-group-chat-contacts (dec max-group-chat-participants))
```

Comment:
`participants` or `contacts` I guess they both represent same thing. It will also might create confusion, better to do this `dec` where it is used or we can find a proper name for this

Reply:
Participants is everyone in the group with the user inclusive, while contacts is the contacts of the user (not inclusive)

Reply:
We could also call it :
- `max-group-chat-external-participants` (seems too verbose)
- `max-group-chat-peers` (peers could mean something else)

Reply:
i would say it's not a constant, it's more like a logic, so i would move `max-group-chat-contacts (dec max-group-chat-participants))` , and in the context it will be more clear , name won't be confusing